### PR TITLE
Charter amendment: Definition of technical project

### PR DESCRIPTION
**Pull request by dthaler**
This PR does the following:
* Add a section that defines Technical Projects per discussion within the charter amendment subcommittee (Thomas Graf, Lisa Caywood, Stephen Walli, and Dave Thaler)
* Replaces other terms with "Technical Projects" where it seemed consistent with the original intent

Note that process-wise this PR requires a governing board vote before it could be merged:

> This Charter may be amended by a 70% vote of the entire Governing Board and BSC, subject to approval by The Linux Foundation.

### CODE REVIEW

File: src/pages/charter.js [hunk 1]
```diff
@@ -243,12 +243,27 @@ const CharterPage = ({ location: {pathname} }) => {
...
         and held by LF Projects, LLC or the Linux Foundation and available for use pursuant to
         LF Projects, LLC’s trademark usage policy, available at <a href="https://www.lfprojects.org/trademarks/" target="_blank" rel="noopener noreferrer">https://www.lfprojects.org/trademarks/</a>.</li>
       </ol>
-      <h2>9) Antitrust Guidelines</h2>
+      <h2>9) Technical Projects</h2>
+      <ol>
+        <li>A Technical Project is defined as being an open source project that meets the following criteria.
+          <ol>
+            <li>It must meet the Trademarks requirement in Section 8.</li>
+            <li>It must accept the eBPF Foundation as its primary governance structure. For example,
```
Comment (lmb) [inline src/pages/charter.js:256]: This sounds like "the eBPF Foundation has the final say" to me. Is that correct? Also, is there a process for a Technical Project to renounce it's status? Vice versa, can the BSC withdraw the Technical Project status?

**Comment by dthaler** [comment]
> The foundation website has moved to https://github.com/ebpffoundation/ebpf.foundation This pull request likely needs to be opened there.

@tgraf I cloned that entire repo but cannot find any charter page in that repo to generate a pull request against.  Can you provide me with a pointer on how to find it?  
@xmulligan closed this, but since I see no way to open a PR there without a charter page I am reopening this issue until I can get instructions on how to open a PR there.

**Comment by dthaler** [comment]
Ok looks like it's not merged yet but is still in https://github.com/ebpffoundation/ebpf.foundation/pull/1

**Comment by xmulligan** [comment]
@dthaler I don't think https://github.com/ebpffoundation/ebpf.foundation is being used to build the website anymore. Linux Foundation moved it to wordpress. Let me connect you with Sridhar on Slack so we can figure this out

**Pull request closed without merging**